### PR TITLE
Ignoring statistics.json

### LINKED ISSUE #24: Rouge `/help` Command Output
https://github.com/ubiquity/bounty-bot/issues/244

### LINKED ISSUE #24: `.gitignore` statistics json
- gitignore will solve issues with merge conflicts
- this will hide it from the files view https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github

### PR DESCRIPTION
**Pull request by jordan-ae**
Resolves [#24](https://github.com/ubiquity/devpool-directory-bounties/issues/24)

### CODE REVIEW

**Comment by 0x4007** [comment]
Please also hide it from the files view

**Comment by jordan-ae** [comment]
> Please also hide it from the files view

I'm sorry but I don't quiet get what you  mean by this. Please can you rephrase thanks 🙏🏽

**Comment by 0x4007** [comment]
There's two bullet points in the issue specification. The second one includes a link to what needs to be implemented.

**Comment by 0x4007** [comment]
I don't think the linguist thing was implemented correctly. I would expect that the GitHub files view would refuse to show the contents when it is implemented correctly.

**Comment by jordan-ae** [comment]
@pavlovcik  What I realized is the files show up in the changed files, but just that the changes in them are not loaded by default and we can click on load diff, if we want, like the way a normal file shows up if it has big changes in it. Also even though they might show up they are not considered as changes by the diffing algorithm so they don't cause merge conflicts. This [article](https://medium.com/@clarkbw/managing-generated-files-in-github-1f1989c09dfd) kind of summaries the expected behavior. Please lmk if I'm thinking it wrong. Thanks

**Review by 0x4007 — APPROVED** [review]
